PERL_POLLUTE removal :-
Allow pilot-link's perl bindings to build with newer Perl.
From Debian ticket 628510

--- bindings/Perl/Makefile.PL.in.orig	Sat Mar 17 19:03:45 2007
+++ bindings/Perl/Makefile.PL.in	Mon Mar 18 22:07:45 2013
@@ -39,10 +39,10 @@ WriteMakefile(
     'VERSION'    => '@PILOT_LINK_VERS@.@PILOT_LINK_MAJOR@.@PILOT_LINK_MINOR@@PILOT_LINK_PATCH@',
     'XSPROTOARG' => '-noprototypes',
     'INC'        => "-I$plincdir",
-    'DEFINE'     => '-DPERL_POLLUTE',
     'PREFIX'     => "$prefix",
-    'INSTALLDIRS'=> 'vendor',
+    'INSTALLDIRS'=> 'site',
     'dynamic_lib'=> {'OTHERLDFLAGS' => $lib},
     'depend'     => {'Pilot.c' => 'const-c.inc const-xs.inc'},
     'clean'      => {'FILES'   => 'const-c.inc const-xs.inc'},
+    'OPTIMIZE'   => "!!CFLAGS!!",
 );
